cumulus/client/consensus/aura/src/collators/slot_based/slot_timer.rs

```diff
@@ -272,13 +272,31 @@ where
 
 	/// Returns the slot and how much time left until the next block production attempt.
 	pub fn time_until_next_block(&mut self, slot_duration: SlotDuration) -> (Duration, Slot) {
-		compute_next_wake_up_time(
+		let (duration, _timestamp_based_slot) = compute_next_wake_up_time(
```

Comment:
```suggestion
		let (duration, timestamp_based_slot) = compute_next_wake_up_time(
```
cause it is used below